lib/gauge_field.cpp

```diff
@@ -75,6 +76,11 @@ namespace quda {
       length = 2*2*nDim*stride*nInternal;  //two comes from being full lattice
     }
 
+    if ((reconstruct == QUDA_RECONSTRUCT_12 || reconstruct == QUDA_RECONSTRUCT_9) &&
```

Comment:
recon 9 -> recon 8

Reply:
thank you 😄

Reply:
That error actually causes the staggered and improved staggered dslash tests to fail, so it's pretty important ;)